### PR TITLE
 add Data Carpentry info,  rm workshop fees

### PR DESCRIPTION
**Pull request by raynamharris**
Address issue https://github.com/ErinBecker/instructor-training-reorg/issues/46

### CODE REVIEW

File: _episodes/22-carpentries.md [hunk 1]
```diff
@@ -95,100 +91,15 @@ Questions, corrections, and additions are *very* welcome.
...
-> workshops include, but are not limited to: illness, injury, family or work related emergencies, weather or transportation malfunction.
-> Instructors may be required to provide appropriate documentation to the Policy Subcommittee. If an instructor is required to reimburse
-> costs, but refuses to do so, or an instructor fails to provide adequate notice of withdrawal more than once, the Carpentries reserve the
-> right to suspend their Carpentry instructor status.  
-> From the [Carpentry Policies repo](https://github.com/carpentries/policies/blob/master/instructor-no-show-policy.md)
-{: .quotation}  
+Policies related to instructor training for both Software Carpentry and Data Carpentry can be found [here](https://github.com/carpentries/policies). Two documents of use are the **[instructor trainers agreement](https://github.com/carpentries/policies/blob/master/trainer-agreement.md)**, the **[code of conduct](https://github.com/carpentries/policies/blob/master/code-of-conduct.md)**, and the **[instructor no show policy](https://github.com/carpentries/policies/blob/master/instructor-no-show-policy.md)**
```
Comment (ChristinaLK) [inline _episodes/22-carpentries.md]: Two -> Three?

File: _episodes/22-carpentries.md [hunk 2]
```diff
@@ -95,100 +91,15 @@ Questions, corrections, and additions are *very* welcome.
...
 
 ### Materials
 
-All of Software and Data Carpentry's lessons materials are freely available
+All of Software Carpentry and Data Carpentry's lessons materials are freely available
```
Comment (ChristinaLK) [inline _episodes/22-carpentries.md]: Minor typo from before, but "lesson materials" not "lessons materials"

**Comment by ChristinaLK** [comment]
This looks good - just a couple quick comments/fixes.

File: _episodes/22-carpentries.md (hunk 1 above)
Comment (ErinBecker) [inline _episodes/22-carpentries.md]: Is the Trainer's agreement relevant here? It refers to Trainers, not Instructors.

**Comment by ErinBecker** [comment]
@raynamharris @ChristinaLK ready to merge after the suggested revisions.

**Comment by raynamharris** [comment]
Hmm. @ChristinaL,K I can't figure out how to make changes to this file to delete the trainers agreement and fix the typo. My access seems to be restricted. What should I do?

**Comment by ChristinaLK** [comment]
Did you delete or rename your fork?

**Comment by raynamharris** [comment]
I deleted my fork already. I think.

File: _episodes/22-carpentries.md (hunk 1 above)
Comment (ChristinaLK) [inline _episodes/22-carpentries.md]: Ah, you're right.  That should probably go.  We don't have an instructors agreement, do we?

**Comment by ChristinaLK** [comment]
I will see if there's anything I can do on my end.

File: _episodes/22-carpentries.md (hunk 1 above)
Comment (ErinBecker) [inline _episodes/22-carpentries.md]: No instructor's agreement yet.

Comment (weaverbel) [inline _episodes/22-carpentries.md]: Mateusz has drafted one though - isn't that right @mkuzak ? I gave feedback on it, but am not sure of the status.

**Comment by ChristinaLK** [comment]
resubmitted as #153

**Pull request closed without merging**